app/src/main/java/com/github/se/polyfit/ui/components/GradientBox.kt

```diff
@@ -0,0 +1,66 @@
+package com.github.se.polyfit.ui.components
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.se.polyfit.ui.theme.*
+
+val EmptyImageVector =
+    ImageVector.Builder(
+            defaultWidth = 0.dp, defaultHeight = 0.dp, viewportWidth = 0f, viewportHeight = 0f)
+        .build()
+
+// Gradient Box builds a white box with gradient border and an optional icon on the top right
+@Composable
+fun GradientBox(
```

Comment:
Looks really cool! It might be a good idea to make the close button optional to improve the reusability of this component.

Reply:
It is already possible to not add in an icon button at the top right, as the default values already set it into an empty icon (if you don't pass anything in for icon, it will just be nothing there)